x-pack/test/functional/services/ml/common_ui.ts

```diff
@@ -438,5 +438,14 @@ export function MachineLearningCommonUIProvider({
         }
       });
     },
+
+    async assertDataTierOptionsVisible(shouldBeVisible: boolean) {
```

Comment:
Although it'll make the name really long, I think this function should mention the date picker to make the name clearer.
e.g. `assertDatePickerDataTierOptionsVisible`

Reply:
Updated here https://github.com/elastic/kibana/pull/181193/commits/35d05bb516722daff98edf7e15701015a2f678b4